app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    todolist = command.split()
+    cmd, file_name, file_copy_name = todolist
+    if cmd == "cp":
+        if file_name != file_copy_name:
```

Comment:
```suggestion
    if cmd == "cp" and file_name != file_copy_name:
```
Use and instead of two ifs